bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -136,25 +136,21 @@ rules:
   - privileged
 - apiGroups: ["k8s.ovn.org"]
   resources:
-  - adminpolicybasedexternalroutes
-  - adminpolicybasedexternalroutes/status
-  - egressfirewalls
-  - egressfirewalls/status
-  - egressqoses
-  verbs:
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups: ["k8s.ovn.org"]
-  resources:
+    - adminpolicybasedexternalroutes
+    - egressfirewalls
     - egressips
+    - egressqoses
```

Comment:
how did egressqoses also lost patch/update rights?

Reply:
does ovnkube-node really do patch/update on egressqos, when i checked this isn't happening. hence removed it.